ui-design/templates/create-template/create-template.md

```diff
@@ -0,0 +1,37 @@
+# Templates
+
+Templates allow the user to create a predefined configuration that can be used to speed up the creation of Virtual Machines.
+
+## Template List
+
+![Template list](img/1-1-templates.png)
+
+From the template list a user can create a new VM from a selected template or create, edit, and delete templates.
+
+## Create Template
+
+![Create template button](img/2-1-create-template.png)
+
+Selecting Create Template will present the user with a wizard similar to the create new VM wizard, but with fewer options.
+
```

Comment:
This has me wondering if the create vm and create template wizard should be the same.  Otherwise we'll need to add new features to two wizards.  This could be the first question you ask the user: "Do you want to create a VM or a VM Template?

Reply:
They are going to be VERY similar. I really like the idea of having a shared component with maybe a few differents depending on the "Type" so that we can reduce the amount of duplicate changes needed.